docs/specs/clients/notify/data-structures.md

```diff
@@ -20,6 +20,10 @@
 
 ```typescript
 {
+  // ID of the notification
+  id: string,
+  // Timestamp when the notification was sent
+  sent_at: string,
```

Comment:
does this match published_at from relay?

Reply:
No this is the created_at timestamp from the Notify Server. Since pages messages can be retrieved directly from there now we need a different field